pkg/metrics/metrics.go

```diff
@@ -0,0 +1,32 @@
+package metrics
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	// Metrics for Controller
+	RBACSyncConfigStatus = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "rbacsync_config_status",
+		Help: "The number of RBACSyncConfigs and RBACSyncClusterConfigs and the status of the processed config",
+	}, []string{"kind", "status"})
+	RBACSyncBindingStatus = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "rbacsync_binding_status",
+		Help: "The number of RoleBindings and ClusterRoleBIndings configured by the controller and their statuses",
+	}, []string{"kind", "status"})
+
+	// Metrics for Mapper/GSuite
+	RBACSyncGsuiteClientCreationStatus = promauto.NewCounter(prometheus.CounterOpts{
+		Name: "rbacsync_gsuite_client_creation_status",
+		Help: "Total number of the status of gsuite client creations",
+	})
+	RBACSyncGsuiteMembersStatus = promauto.NewCounterVec(prometheus.CounterOpts{
+		Name: "rbacsync_gsuite_members_status",
+		Help: "Total number of the status of calls to gsuite with labels for state",
+	}, []string{"status"})
+	RBACSyncGsuiteMembersLatency = promauto.NewHistogram(prometheus.HistogramOpts{
+		Name: "rbacsync_gsuite_members_latency",
```

Comment:
Prometheus best practice here would to have this suffixed with `duration_seconds`.

Reply:
updated, thanks for reviewing